src/site/content/en/blog/community-highlight-miriam-suzanne/index.md

```diff
@@ -0,0 +1,75 @@
+
+**Miriam:** Yes, for some people–but there's no universal answer here. Take variables for example. Sass variables are lexically scoped, and compiled on the server, with organized data structures like lists and objects, color manipulation, etc. That's great for design system logic that doesn't need to run in the browser. CSS Variables have some overlap, they can also store values, but they provide an entirely different set of cascade-based strengths and weaknesses. Sass can't handle custom properties, and CSS can't really handle structured data. Both are useful, and both are powerful–but your specific needs may vary. 
+
+I think it's great when people can eliminate tools that they no longer need, and some projects might not require both server- and client-side variables. Wonderful! But it's too simple to assume that means they are identical, and one simply replaces the other. There will always be use-cases for some design logic to happen server-side, and some to happen client-side–even if we get to the point where the languages provide essentially the same features. Pre-processors are with us for the long term.
+
+**Rachel:** Is there anything that has surprised you as you’ve got more involved in the work of creating standards, or anything that you think folk generally might not be aware of about the process?
+
+**Miriam:** Before getting involved, the standards process felt like a mysterious and magical black box, and I wasn't sure what to expect. I was scared that I might not have enough knowledge of browser-internals to contribute, but the reality is they don't need more browser engineers–they need more developers and designers who are building websites and applications in the wild. 
+
+I was surprised to find that very few of the people involved are primarily focused on standards, but also very few of them are primarily developing or designing websites. The W3C is made up of 'volunteers' from member organizations–often paid by those orgs, but not as their primary job–and membership isn't cheap. That skews the participants away from day-to-day designers and developers, especially those of us doing client work at small agencies, or freelance. My role as an Invited Expert would be totally volunteer, an expensive hobby, if I hadn't found outside funding for that work.
+
+In reality, the process is quite open and public and needs developer involvement–but there are always so many conversations happening at once, it can be hard to find your place. Especially if it's not your day job.
+
+{% Aside %}
+To find out how to contribute to the web platform, check out the [Web Platform Contribution Guide](https://wpc.guide/).
+{% endAside %}
+
+**Rachel:** Container Queries have been the holy grail for a lot of web developers for many years. I’m very excited about the fact that we’re getting them. I feel as if a lot of folk are thinking about the utility of Container Queries–do you feel they have potential to unlock more creativity too?
```

Comment:
```suggestion
**Rachel:** Container queries have been the holy grail for a lot of web developers for many years. I’m very excited about the fact that we’re getting them. I feel as if a lot of folk are thinking about the utility of container queries&mdash;do you feel they have potential to unlock more creativity too?
```

Reply:
Spelling seems to vary in places, but is most consistently lower case, so I've adjusted accordingly.